src/components/Drawer/index.vue

```diff
@@ -0,0 +1,289 @@
+
+      // Detail 中
+      &.content {
+        margin-right: 0;
+      }
+
+      .form-buttons {
+        //position: absolute;
+        // bottom: 13px;
+        // margin-left: 20%;
+        // margin-top: 0;
+      }
+
+      // Form 中的子 form
+      .el-form {
+        margin-left: 0;
+        margin-top: 0;
+        margin-bottom: 0;
+      }
+
+      .el-form-item {
+        .el-form-item__label {
+          padding-right: 20px;
+        }
+
+        .el-radio {
+          line-height: 25px;
+          margin-right: 13px;
+
+          .el-radio__label {
+            padding-left: 5px;
+          }
+        }
+      }
+
+      &.el-form--label-top {
+        .el-radio-group {
+          .el-radio {
+            display: block;
+            padding-bottom: 3px;
+          }
+        }
+
+        .el-form-item {
+          padding-left: 12px;
+
+          .el-form-item__label {
+            padding: 0 20px 0 0;
+            line-height: 30px;
+          }
+
+          .sub-form {
+            margin-left: -1px;
+
+            .form-fields {
+              max-height: unset;
+            }
+          }
+        }
+
+        &.form-fields {
+          //overflow: auto;
+          //max-height: calc(100vh - 180px);
+        }
+
+        .el-checkbox-group {
+          .el-checkbox {
+            display: block;
+            padding-bottom: 3px;
+          }
+        }
+
+        .el-form-item__content:has(.el-checkbox):not(:has(.el-checkbox-group)) {
+          display: inline-block; /* 更改为 inline-block */
+          //width: unset; /* 这个设置上去后，平台详情中， Automations 会有问题 */
+          vertical-align: bottom;
+        }
+
+        .el-form-item__content {
+          form {
+            .el-form-item {
+              padding-left: 0;
+            }
+          }
+        }
+      }
+
+      .form-group-header {
+        margin-left: 20px;
+      }
+    }
+
+    .el-drawer__header {
+      border-bottom: 1px solid #EBEEF5;
+      margin-bottom: 0;
+      padding: 15px 20px;
+      font-size: 16px;
+      font-weight: 500;
+      color: var(--color-text-primary);
+    }
+
+    .sql.container {
+      display: none;
+    }
+
+    .page {
+      overflow-y: auto;
+
+      height: calc(100vh - 110px);
+
+      &.tab-page {
+        .page-content {
+          padding-right: 0;
+          padding-left: 0;
+        }
+      }
+
+      .page-content {
+        height: unset;
+        padding-right: 10px;
+        padding-left: 20px;
+
+        & > div {
+          margin-bottom: 1px;
+        }
+      }
+
+      .ibox {
+        margin-bottom: 10px;
+        border: none;
+      }
+    }
+
+    .drawer__content, .tab-page-content {
+      background: #f3f3f3;
+    }
+
+    .drawer__footer {
+      border-top: solid 1px #f3f3f3;
+    }
+
+    //.el-drawer__header {
+    //  margin-bottom: 20px;
+    //
+    //  span {
+    //    font-size: 16px;
+    //    font-weight: 800;
+    //    color: var(--color-text-primary);
+    //  }
+    //}
+  }
+}
+</style>
```

Comment:
This is not a JavaScript programming problem, but rather CSS styling based on HTML elements provided using the vue.js framework. So there are very few issues that need to be addressed or optimized.

However, if you find anything specific and non-trivial about the CSS properties of certain selectors (like .drawer or @page), feel free to ask!